src/store/transaction/transaction.thunks.js

```diff
@@ -462,6 +452,9 @@ function exit(amount, account, fee) {
     const {
       global: { wallet, coordinatorStateTask },
     } = getState();
+
+    dispatch(transactionActions.startTransactionSigning());
```

Comment:
Signing? An exit is an L2 tx

Reply:
I'm reusing that function to be aware in the view if the transaction fail and reactivate the btn. Maybe I can rename that action to startTransaction and endTransaction